### PR TITLE
Create and Remove

### PR DESCRIPTION
**Pull request by AymenBida**
In this milestone I did: 

- Implement the createBook action 
- Implement the removeBook action

### CODE REVIEW

File: src/containers/BooksForm.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            {CATEGORIES.map(
+              (category) => <option key={category.id} value={category.id}>{category.name}</option>,
+            )}
+          </select>
+        </label>
+      </div>
+      <div>
+        <button type="button" onClick={handleSubmit}>Submit</button>
+      </div>
+    </form>
```
Comment (Berabjesus) [inline src/containers/BooksForm.js]: - [ ] Lets make the button type `submit` `<button type="button" onClick={handleSubmit}>Submit</button>` and add ` <form onSubmit={handleSubmit}>` to the form.

File: src/containers/BooksForm.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  const handleChange = (event) => {
+    setBook({ ...book, [event.target.id]: event.target.value });
+  };
+
+  const handleSubmit = () => {
+    dispatch(createBook(book));
+    setBook({ title: '', category: book.category });
+    window.idCounter += 1;
+  };
```
Comment (Berabjesus) [inline src/containers/BooksForm.js]: - [x] Add `event.preventDefault();` before dispatch to prevent page refresh after the submit button is clicked.

**Review by Berabjesus — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 
Hey @AymenBida excellent job on this milestone. You have written a clean code that does the job.
### However, there is one minor change you need to make.

- [ ] `BookForm` component should implement `handleSubmit` and attaches it to **form's** `onSubmit` event [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleSubmit.png). Although your code does the job well its best to handle form submission in a different way since its handling a submit action, not a click action. Check the inline comment below.

#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Review by afanimartin — APPROVED** [review]
## Approved ✅ ✅ ✅ 
Nice work making the requested change. Merge and proceed to the next milestone.


### Martin Afani
#### Slack: whiz25

**Pull request closed without merging**